src/ens.ts

```diff
@@ -248,32 +242,32 @@ export default class Ens extends NamingService {
   /**
    * This was done to make automated tests more configurable
    */
-  private resolverCallToName(resolverContract, nodeHash) {
-    return this.callMethod(resolverContract.methods.name(nodeHash));
+  private resolverCallToName(resolverContract: Contract, nodeHash) {
+    return this.callMethod(() => resolverContract.fetchMethod('name', [nodeHash]) );
```

Comment:
It might be more readable to have it like this `this.callMethod(resolverContract, 'name', nodeHash)`